gramine/README.md

```diff
@@ -0,0 +1,6 @@
+## How to build your Gramine application
+
+```shell
+docker build -t gramine-tee-worker-post-compute:latest -f gramine/Dockerfile  .
```

Comment:
Do we change the image name to be closer to the latest convention ?

Reply:
Yes, is https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/74/commits/dfb01e7f6e4ae4e8a395983938ded111b0b26ae1 fine for you?

Reply:
Thanks !